common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -117,10 +137,18 @@ function (Sjson, AsyncProcess) {
                 .on(events, 'li[data-index]', this.onCaptionHandler);
 
             if (this.showLanguageMenu) {
+                this.languageChooserEl.on({
+                    keydown: this.handleKeypress
+                }, '.language-menu');
+
+                this.languageChooserEl.on({
+                    keydown: this.handleKeypressLink
+                }, '.control-lang');
+
                 this.container.on({
                     mouseenter: this.onContainerMouseEnter,
-                    mouseleave: this.onContainerMouseLeave
-                });
+                    mouseleave: this.onContainerMouseLeave,
+                })
```

Comment:
I'm not sure,  I checked out the [jshint diff quality report](https://build.testeng.edx.org/job/edx-platform-quality-pr/6373/Diff_Quality_Report/) and it didn't pick up anything.  This file was listed in the report too, so I think it was checked.